### PR TITLE
Data generation

### PR DESCRIPTION
**Pull request by StevenPurn**
Adding script to generate data for Mongo & Postgres.

### CODE REVIEW

File: RememberThese.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+mongoimport --jsonArray -d apatight -c restaurants2  --file places.json --numInsertionWorkers 8
```
Comment (send-on) [inline RememberThese.txt:1]: You can increase the # of workers to make it faster.  Try 16 and see if it makes a difference.  Not sure if the # of workers === # of cores but using 16 helped mine.

File: create10mEntries.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    rating: faker.random.number(1, 5),
+    relative_time_description: faker.date.recent(Math.floor(Math.random() * 7)),
+    text: faker.lorem.sentence(),
+    place_id: index,
+  };
+  return review;
+};
+
+const createReviews = (count) => {
+  const reviews = [];
```
Comment (send-on) [inline create10mEntries.js:21]: Should this be const or let?

File: create10mEntries.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      if (i === 1) {
+        stream.write(`[${JSON.stringify(entryFactory(i))}`);
+      } else {
+        ok = stream.write(`,${JSON.stringify(entryFactory(i))}`);
+      }
+    } while (i < count && ok);
+    if (i < count) {
+      stream.once('drain', write);
+    } else {
+      stream.write(']');
```
Comment (send-on) [inline create10mEntries.js:62]: Smart!  I might use this method.   Any issues with async?

File: package.json [hunk 4]
```diff
@@ -14,8 +14,9 @@
     "start": "node ./server/server.js",
     "seed": "node ./seedDb.js",
     "docker": "npm run build:prod && npm run seed & npm run start",
-    "lint" : "eslint .",
-    "lint:fix" : "eslint . --fix"
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "fix": "truncate -s -1 dummydata2.json && echo ] >> dummydata2.json"
```
Comment (send-on) [inline package.json:19]: Is this still needed with the closing ] in your seed generation script?

**Review by send-on — APPROVED** [review]
Looks good.

File: package.json (hunk 4 above)
Comment (StevenPurn) [reply package.json:19]: No, I didn't even realize it was still there.

File: create10mEntries.js (hunk 2 above)
Comment (StevenPurn) [reply create10mEntries.js:21]: const, editing contents of an array is not reassigning.

File: create10mEntries.js (hunk 3 above)
Comment (StevenPurn) [reply create10mEntries.js:62]: No, it adds it to the end of the buffer so it works as expected.

File: RememberThese.txt (hunk 1 above)
Comment (StevenPurn) [reply RememberThese.txt:1]: Thanks, will try!

**Pull request merged**